libert/applications/ert_tui/enkf_tui_export.c

```diff
@@ -601,10 +388,6 @@ void enkf_tui_export_menu(void * arg) {
   menu_add_separator(menu);
   menu_add_item(menu , "Export P( a =< x < b )"                                 , "sS" , enkf_tui_export_fieldP , enkf_main , NULL);
   menu_add_separator(menu);
-  menu_add_item(menu , "Export cell values to text file(s)"                     , "cC" , enkf_tui_export_cell    , enkf_main , NULL);
-  menu_add_item(menu , "Export line profile of a field to text file(s)"         , "pP" , enkf_tui_export_profile , enkf_main , NULL);
-  menu_add_item(menu , "Export time development in one cell to text file(s)"    , "tT" , enkf_tui_export_time    , enkf_main , NULL);
-  menu_add_separator(menu);
```

Comment:
so these functionalities are entirely removed?

Reply:
Yes they are. I am 99% certain they are never used - and if people want the functionality we can help them implement a Python script.